internal/oci/runtime_oci.go

```diff
@@ -635,14 +635,42 @@ func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration,
 		if err := kill(pid); err != nil {
 			return fmt.Errorf("failed to kill process: %v", err)
 		}
+	case newTimeout := <-c.stopTimeoutChan:
```

Comment:
If we are in the kill phase then we can ignore the update.

Reply:
added!